src/lib/components/load_failure_view.tsx

```diff
@@ -1,19 +1,17 @@
-import React from 'react'
+import * as React from 'react'
 import {
   requireNativeComponent,
 } from 'react-native'
 
-export default class LoadFailureView extends React.Component {
+export default class LoadFailureView extends React.Component<any,any> {
+  static propTypes = {
+    // A callback that is called when the user requests a retry.
+    onRetry: React.PropTypes.func,
+  }
```

Comment:
We need to have propTypes defined for native components but yes this should be defined in the Props interface as well